tests/test_fs.py

```diff
@@ -79,11 +81,14 @@ def test_p():
     fs = XRootDPyFS("root://eosuser.cern.ch//eos/user/")
     assert fs._p("./") == "//eos/user"
     assert fs._p("l") == "//eos/user/l"
-    assert fs._p("/eos") == "//eos"
```

Comment:
This `assert` does not make much sense to me: it tests a path that is parent to the mounted one, so it should never happen.